src/fides/core/config/redis_settings.py

```diff
@@ -11,19 +11,48 @@
 class RedisSettings(FidesSettings):
     """Configuration settings for Redis."""
 
-    host: str = "redis"
-    port: int = 6379
-    user: Optional[str] = ""
-    password: str = "testpassword"
-    charset: str = "utf8"
-    decode_responses: bool = True
-    default_ttl_seconds: int = 604800
-    identity_verification_code_ttl_seconds: int = 600
-    db_index: Optional[int]
-    enabled: bool = False
-    ssl: bool = False
-    ssl_cert_reqs: Optional[str] = "required"
-    connection_url: Optional[str] = None
+    charset: str = Field(default="utf8", description="TODO")
+    connection_url: Optional[str] = Field(
+        default=None,
+        description="A full connection URL to the Redis cache. If not specified, this URL is automatically assembled from the host, port, password and db_index specified above.",
+    )
+    db_index: Optional[int] = Field(
+        default=None,
+        description="The application will use this index in the Redis cache to cache data.",
+    )
+    decode_responses: bool = Field(default=True, description="")
+    default_ttl_seconds: int = Field(
+        default=604800,
+        description="The number of seconds for which data will live in Redis before automatically expiring.",
+    )
+    enabled: bool = Field(
+        default=False,
+        description="Whether the application's Redis cache should be enabled. Only set to false for certain narrow uses of the application.",
+    )
+    host: str = Field(
+        default="redis",
+        description="The network address for the application Redis cache.",
+    )
+    identity_verification_code_ttl_seconds: int = Field(default=600, description="TODO")
```

Comment:
Help

Reply:
Sets TTL for cached identity verification code as part of subject requests.